lib/webtask.js

```diff
@@ -708,6 +736,92 @@ exports.ensure_authenticated = function(ctx) {
             });
         }
 
+        function check_against_secret(jwt, callback) {
+            try {
+                auth_token = req.headers['authorization']
+                .match(/\s.+$/)[0]
```

Comment:
I think the standard is to require that the header format be `Authorization: Bearer <token>`. We should ensure that the incoming header conforms to this standard.